service_mesh/v2x/ossm-route-migration.adoc

```diff
@@ -0,0 +1,15 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ossm-route-migration"]
+= Route Migration
+include::_attributes/common-attributes.adoc[]
+:context: route-migration
+
+toc::[]
+
+Automatic route creation, also known as Istio OpenShift Routing (IOR), is a deprecated feature that is disabled by default for any `ServiceMeshControlPlane` resource that was created using {SMProductName} 2.5 and later versions. Migrating from IOR to explicitly-managed routes provides a more flexible way to manage and configure ingress gateways. When route resources are explicitly created they can be managed alongside the other gateway and application resources as part of a GitOps management model.
+
+include::modules/ossm-migrating-from-ior-to-explicitly-managed-routes.adoc[leveloffset=+1]
+
+.Additional resources
```

Comment:
Same comment about this section as in the other assembly.

If it's for the assembly, this is how it should be formatted:
```suggestion
[role="_additional-resources"]
[id="additional-resources_{context}"]
== Additional resources
```

If it's just meant for the module above it, then all it needs is a role attribute:
```suggestion
[role="_additional-resources"]
.Additional resources
```

Reply:
updated for assembly. done.